terminal.js

```diff
@@ -1,25 +1,29 @@
 /* terminal.js by Cassidy James <c@ssidyjam.es> for System76.com */
+$default_delay = 200;
```

Comment:
We use camelCase throughout the rest of the code, should probably use that here for consistency.